pkg/cri/server/sandbox_run.go

```diff
@@ -386,6 +397,13 @@ func (c *criService) RunPodSandbox(ctx context.Context, r *runtime.RunPodSandbox
 	}); err != nil {
 		return nil, fmt.Errorf("failed to update sandbox status: %w", err)
 	}
+	sandboxStatus := sandbox.Status.Get()
+	span.AddEvent("Successfully started sandbox container",
+		tracing.EventAttribute(
+			attribute.Int("sandbox.status.pid", int(sandboxStatus.Pid)),
```

Comment:
`tracing.Attribute("sandbox.status.pid", int(sandboxStatus.Pid))`
will pass this as int attribute, right? (no need to have `attribute.Int` here).